### PR TITLE
feature: support to relocate/rename assets deduplicated by origin URL

### LINKED ISSUE #77: Relocate assets
With the new "deduplicate by URL" feature for assets, it's possible for them to be in the system already, just not be at the path we expect tp place them.

We should add support to relocate them, similar to what's done for objects.

### PR DESCRIPTION
**Pull request by dkarlovi**
Closes #77.

### CODE REVIEW

File: src/ImportDefinitionsBundle/Resources/public/pimcore/js/interpreters/asseturl.js [hunk 1]
```diff
@@ -66,6 +66,18 @@ pimcore.plugin.importdefinitions.interpreters.asset_url = Class.create(pimcore.p
             fieldLabel: t('importdefinitions_interpreter_asset_url_deduplicate_by_url'),
             name: 'deduplicate_by_url',
             value : Ext.isDefined(config.deduplicate_by_url) ? config.deduplicate_by_url : false
+        },
+        {
+            xtype : 'checkbox',
+            fieldLabel: t('importdefinitions_relocate_existing_objects'),
+            name: 'relocate_existing_objects',
+            value : config.deduplicate_by_url && Ext.isDefined(config.relocate_existing_objects) ? config.relocate_existing_objects : false
```
[the remark this answers is not in the document]
Comment (dpfaffenbauer) [reply src/ImportDefinitionsBundle/Resources/public/pimcore/js/interpreters/asseturl.js]: yes, can be done using listeners with extjs. can you change that?

[the remark this answers is not in the document]
Comment (dkarlovi) [reply src/ImportDefinitionsBundle/Resources/public/pimcore/js/interpreters/asseturl.js]: Sure, need to check how it's done in ExtJS.

File: src/ImportDefinitionsBundle/Resources/public/pimcore/js/interpreters/asseturl.js [hunk 2]
```diff
@@ -15,6 +15,23 @@ pimcore.registerNS('pimcore.plugin.importdefinitions.interpreters.asset_url');
 
 pimcore.plugin.importdefinitions.interpreters.asset_url = Class.create(pimcore.plugin.importdefinitions.interpreters.abstract, {
     getLayout : function (fromColumn, toColumn, record, config) {
+        var deduplicateByUrlEnabled = Ext.isDefined(config.deduplicate_by_url) ? config.deduplicate_by_url : false
```
Comment (dkarlovi) [inline src/ImportDefinitionsBundle/Resources/public/pimcore/js/interpreters/asseturl.js]: I'm not in love with this code, TBH, it looks quite verbose for what it does.

Comment (dpfaffenbauer) [reply src/ImportDefinitionsBundle/Resources/public/pimcore/js/interpreters/asseturl.js]: yep.... nothing we can do about it.

**Pull request merged**